sdk/core/Azure.Core/api/Azure.Core.net6.0.cs

```diff
@@ -17,7 +17,9 @@ public abstract partial class AsyncPageable<T> : System.Collections.Generic.IAsy
     }
     public static partial class AzureCoreExtensions
     {
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("This class utilizes reflection-based JSON serialization and deserialization which is not compatible with trimming.")]
```

Comment:
```suggestion
        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("This method utilizes reflection-based JSON serialization and deserialization which is not compatible with trimming.")]
```

(nit) since this attribute is on a method, not a class.

Reply:
Ah thanks for catching that, I'll fix it!